src/main/java/net/coderbot/iris/uniforms/CommonUniforms.java

```diff
@@ -63,10 +63,8 @@ public static void addCommonUniforms(UniformHolder uniforms, IdMap idMap) {
 			.uniform1f(PER_FRAME, "screenBrightness", () -> client.options.gamma)
 			.uniform1f(PER_TICK, "playerMood", CommonUniforms::getPlayerMood)
 			.uniform2i(PER_FRAME, "eyeBrightness", CommonUniforms::getEyeBrightness)
-			// TODO: This should be smoothed, but not smoothing it is better than nothing.
 			.uniform2i(PER_FRAME, "eyeBrightnessSmooth", new SmoothedVec2f(10.0f, CommonUniforms::getEyeBrightness))
 			.uniform1f(PER_TICK, "rainStrength", CommonUniforms::getRainStrength)
-			// TODO: This should be smoothed, but not smoothing it is better than nothing.
```

Comment:
This still isn't resolved, the wetness value isn't actually smoothed like it should be yet.

Reply:
Oh okay. Now I know